ui/src/layout/ActivityPanel.js

```diff
@@ -150,6 +204,37 @@ const ActivityPanel = () => {
               </IconButton>
             </Tooltip>
           </CardActions>
+          <Divider />
+          <Typography className={classes.heading}>
+            {translate('activity.nowPlaying')}
+          </Typography>
+          <List className={classes.list}>
+            {nowPlaying.map((user) => {
+              return (
+                <ListItem key={user.coverArtId}>
+                  <ListItemAvatar>
+                    <Avatar
+                      alt={`${user.title} ${translate('activity.coverArt')}`}
+                      src={subsonic.getCoverArtUrl(user)}
+                    />
+                  </ListItemAvatar>
+                  <ListItemText
+                    className="primary"
+                    primary={`${user.username}`}
+                    secondary={`${user.title} - ${user.artist}`}
+                  />
+                  <ListItemSecondaryAction>
+                    <ListItemText
+                      primary={`${user.minutesAgo} ${translate(
+                        'activity.minutesAgo'
+                      )}`}
+                      secondary="place-holder"
```

Comment:
we need to add a "placeholder" to not break material UI CSS, i.e if we don't add a placeholder than css breaks for material UI like the following. we add visibility hidden to the div which contains the "place-holder" text 

![Screenshot from 2021-04-13 15-00-27](https://user-images.githubusercontent.com/54576074/114530904-02f45c00-9c69-11eb-8ffa-06be6a94e644.png)